spec/gitlab/client/commits_spec.rb

```diff
@@ -46,6 +46,24 @@
     end
   end
 
+  describe '.commit_refs' do
+    before do
+      stub_get('/projects/3/repository/commits/0b4cd14ccc6a5c392526df719d29baf4315a4bbb/refs', 'project_commit_refs')
+      @refs = Gitlab.commit_refs(3, '0b4cd14ccc6a5c392526df719d29baf4315a4bbb')
+    end
+
+    it 'gets the correct resource' do
+      expect(a_get('/projects/3/repository/commits/0b4cd14ccc6a5c392526df719d29baf4315a4bbb/refs'))
+        .to have_been_made
+    end
+
+    it 'returns an Array of refs' do
+      expect(@refs.map(&:to_h))
+        .to include('type' => 'branch', 'name' => '12-1-stable')
+        .and include('type' => 'tag', 'name' => 'v12.1.0')
```

Comment:
I would've preferred using a single `include` call here, but RuboCop complained about "unnecessary" curly braces around the second Hash:

```
spec/gitlab/client/commits_spec.rb:63:9: C: Style/BracesAroundHashParameters: Redundant curly braces around a hash parameter.
        { 'type' => 'tag', 'name' => 'v12.1.0' }
        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
```